packages/core/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { DefaultPubSubTopic } from "./lib/constants.js";
+export { DefaultPubSubTopic } from "@waku/utils/constants";
```

Comment:
Why moving the value? It's not really a `utils` value. Remaining in `@waku/core` seems fair.